sdk-extensions/jfr-events/build.gradle

```diff
@@ -8,7 +10,9 @@ ext.moduleName = 'io.opentelemetry.sdk.extension.jfr'
 dependencies {
     implementation project(':opentelemetry-api'),
             project(':opentelemetry-sdk')
-    implementation libraries.guava
+
+    compileOnly libraries.weak_lock_free
```

Comment:
How do I ensure that we have this library around when loading JFR extension? Expect that opentelemetry is loaded and reachable and pulls in this dependency?

Reply:
Yeah this has the pattern where we vendor into `opentelemetry-context`, which is always pulled in as a dependency.

Reply:
By the way if there's a better pattern I'm happy to try it :) This is the only one I could come up with without duplicating the class into each artifact that uses it.